Autocomplete/predefined-items-and-recent-searches/src/index.js

```diff
@@ -0,0 +1,64 @@
+import algoliasearch from "algoliasearch/lite";
+import {
+  autocomplete,
+  getAlgoliaHits,
+  reverseHighlightItem,
+} from "@algolia/autocomplete-js";
+import { predefinedItems } from "./predefined-items";
+import { recentSearches } from "./recent-searches";
+
+const searchClient = algoliasearch(
+  "latency",
+  "6be0576ff61c053d5f9a3225e2a90f76"
+);
+
+autocomplete({
+  container: "#autocomplete",
+  plugins: [predefinedItems, recentSearches],
```

Comment:
For now, the order is always `plugins` -> `getSources` here.
In order to display the predefined items before recent searches, I created a plugin for the predefined items.
Does it make sense to you?

Reply:
I think it could be fixed if a plugin would expose sources instead. Is there a reason we didn't go with that design?

Reply:
If you look at [this custom plugin](Autocomplete/predefined-items-and-recent-searches/src/predefined-items.js), a plugin is actually an object with `getSources` function. If you put like in `plugins` array, the autocomplete-core takes care of calling the sources, and some other lifecycle hooks.
It means if user wants, they can do things like:

```js
autocopmlete({
  ...
   getSources(params) {
    return recentSearches.getSources(params);
  }
})
```